### PR TITLE
If VBMS is down, return a helpful error (Close: #25)

### PR DESCRIPTION
**Pull request by paultag**
This puts the error less on the user, and more on VBMS. Retrying the
same page (by refreshing or clicking the href) should restart the
caseflow workflow, since we're not sure the state that things have been
left in.

### CODE REVIEW

File: config/initializers/connect_vbms.rb [hunk 1]
```diff
@@ -2,4 +2,18 @@
...
 require 'vbms'
 
-$vbms = VBMS::Client.from_env_vars(env_name: ENV["CONNECT_VBMS_ENV"])
+class CaseflowLogger
+  def log(event, data)
+    case event
+    when :request
+        if data[:response_code] != 200
+            Rails.logger.info "Serverside failure! Server response: " + data[:response_body]
+        end
```
Comment (alex) [inline config/initializers/connect_vbms.rb]: We're using 2 space indents.

Comment (paultag) [inline config/initializers/connect_vbms.rb]: gah, I need to reconfigure vim

File: config/initializers/connect_vbms.rb [hunk 2]
```diff
@@ -2,4 +2,18 @@
...
+class CaseflowLogger
+  def log(event, data)
+    case event
+    when :request
+      if data[:response_code] != 200
+        Rails.logger.info "Serverside failure! Server response: " + data[:response_body]
+      end
+    end
+  end
+end
```
Comment (alex) [inline config/initializers/connect_vbms.rb]: Can we put this logging in a separate pr for #21

Comment (paultag) [inline config/initializers/connect_vbms.rb]: Sure, I'll split that out now

**Comment by paultag** [comment]
Logging split off into #31 - needs re-review!
